ecosystem/sep-0031.md

```diff
@@ -351,15 +356,17 @@ Example response:
 
 If the transaction cannot be found, the endpoint should return a `404 NOT FOUND` result.
 
-#### Pending info update
+#### Pending customer info update
 
-In certain cases the receiver might need to request updated information, for example if the bank tells them that the provided receiver name is incorrect, or missing a middle initial.  In this case, the transaction should go into the `pending_info_update` state until the sender provides updates.
+In certain cases the receiver might need to request updated information.  For example, if the bank tells them that the provided receiver name is incorrect, or missing a middle initial.  Since this information was sent via SEP-12, the transaction should go into the `pending_customer_info_update` state until the sender makes another `PUT /customer` request.
```

Comment:
How does the client know which fields need to be updated?

Reply:
The `/transaction` response status should be `pending_customer_info_update`, so they'll know an update is needed. They'll make an additional request to `GET /customer` to see which fields are still needed.

Reply:
Updated to include how the client determine which fields need updating.